ML project/Customer_churn_prediction.py

```diff
@@ -0,0 +1,512 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+get_ipython().run_line_magic('matplotlib', 'inline')
+
+
+# In[2]:
+
+
+df = pd.read_csv('train.csv')
+
+
+# In[3]:
+
+
+df.head()
+
+
+# In[4]:
+
+
+df.info()
+
+
+# In[5]:
+
+
+df.var()
+
+
+# Looks like 4 columns have high variance and would possibly benefit some kind of normalization
+
+# In[6]:
+
+
+sns.heatmap(df.corr(),cmap='BuGn');
```

Comment:
@Ettore1991 I would suggest to go for scatter plots as well to eyeball non-linear pattern. Humans are good up to 2D, let's exploit it